documentation/book/con-what-the-topic-operator-does.adoc

```diff
@@ -27,7 +25,7 @@ And also, in the other direction:
 * if a topic is deleted from the Kafka cluster, the operator will create the `KafkaTopic` describing it
 * if a topic in the Kafka cluster is changed, the operator will update the `KafkaTopic` describing it
 
-This is beneficial to a {ProductPlatformName}-centric style of deploying applications because it allows you to declare a `KafkaTopic` as part of your application's deployment and the operator will take care of creating the topic for you.
+This is beneficial to an {ProductPlatformName}-centric style of deploying applications because it allows you to declare a `KafkaTopic` as part of your application's deployment and the operator will take care of creating the topic for you.
```

Comment:
Upstream, this renders to _OpenShift or Kubernetes-centric_ ... is that ok?

Reply:
@ncbaratta?

Reply:
I'd remove the 'centric' part.  Maybe "This is beneficial to an {ProductPlatformName} style of deploying applications..."

Downstream we'll just change the value of this attribute to 'OpenShift' :)